docs/docs/recipes/integrate-logto/go-web.mdx

```diff
@@ -0,0 +1,347 @@
+		)
+
+		// Use Logto to control the content of the home page
+		authState := "You are not logged in to this website. :("
+
+		if logtoClient.IsAuthenticated() {
+			authState = "You are logged in to this website! :)"
+		}
+
+		homePage := `<h1>Hello Logto</h1>` +
+			"<div>" + authState + "</div>"
+
+		ctx.Data(http.StatusOK, "text/html; charset=utf-8", []byte(homePage))
+	})
+
+	// ...
+}
+```
+
+## Sign In
+
+<SignInFlowSummary />
+
+### Configure Redirect URI
+
+<ConfigureRedirectUri
+  figureSrc={redirectUriFigure}
+  redirectUri="http://localhost:8080/sign-in-callback"
+/>
+
+### Add a route for handling sign-in requests
+
+```go
+//main.go
+func main() {
+	// ...
+
+	// Add a link to perform a sign-in request on the home page
+	router.GET("/", func(ctx *gin.Context) {
+		// ...
+		homePage := `<h1>Hello Logto</h1>` +
+			"<div>" + authState + "</div>" +
+			// Add link
+			`<div><a href="/sign-in">Sign In</a></div>`
+
+		ctx.Data(http.StatusOK, "text/html; charset=utf-8", []byte(homePage))
+	})
+
+	// Add a route for handling sign-in requests
+	router.GET("/sign-in", func(ctx *gin.Context) {
+		session := sessions.Default(ctx)
+		logtoClient := client.NewLogtoClient(
+			logtoConfig,
+			&SessionStorage{session: session},
+		)
+
+		// The sign-in request is handled by Logto.
+		// The user will be redirected to the Redirect URI on signed in.
+		signInUri, err := logtoClient.SignIn("http://localhost:8080/sign-in-callback")
+		if err != nil {
+			ctx.String(http.StatusInternalServerError, err.Error())
+			return
+		}
+
+		// Redirect the user to the Logto sign-in page.
+		ctx.Redirect(http.StatusTemporaryRedirect, signInUri)
+	})
+
+	// ...
+}
+```
+
+### Add a route for handling sign-in callback requests
+
+When the user signs in successfully on the Logto sign-in page, Logto will redirect the user to the Redirect URI.
+
+Since the Redirect URI is `http://localhost:8080/sign-in-callback`, we add the `/sign-in-callback` route to handle the callback after signing in.
+
+```go
+// main.go
+func main() {
+	// ...
+
+	// Add a route for handling sign-in callback requests
+	router.GET("/sign-in-callback", func(ctx *gin.Context) {
+		session := sessions.Default(ctx)
+		logtoClient := client.NewLogtoClient(
+			logtoConfig,
+			&SessionStorage{session: session},
+		)
+
+		// The sign-in callback request is handled by Logto
+		err := logtoClient.HandleSignInCallback(ctx.Request)
+		if err != nil {
+			ctx.String(http.StatusInternalServerError, err.Error())
+			return
+		}
+
+		// Jump to the page specified by the developer.
+		// This example takes the user back to the home page.
+		ctx.Redirect(http.StatusTemporaryRedirect, "/")
+	})
+
+	// ...
+}
+```
+
+## Sign out
+
+### Configure Post Sign-out Redirect URI
+
+<ConfigurePostSignOutRedirectUri
+  figureSrc={postSignOutRedirectUriFigure}
+  postSignOutRedirectUri="http://localhost:8080"
+/>
+
+This configuration enables the user to return to the home page after signing out.
```

Comment:
```suggestion
After signing out, it'll be great to redirect your user back to your website. Let's add `http://localhost:8080` as one of the Post Sign-out URIs in Admin Console (shows under Redirect URIs), and use the URL as the parameter when calling `.SignOut()`.
```

Reply:
This part is described in the `ConfigPostSignOutRedirectUri` component.
I think it's redundant here.